main.ts

```diff
@@ -86,52 +87,6 @@ function createWindow() {
 
 }
 
-// Edited version of a template menu-bar provided by the electron API,
-// refer to https://electronjs.org/docs/api/menu for more information.
-const mainMenuTemplate: Electron.MenuItemConstructorOptions[] = [
-  {
-    label: 'File',
-    submenu: [
-      {
-        label: 'Quit CATcher', accelerator: 'CmdOrCtrl+Q', click() { app.quit(); }
-      }
-    ]
-  },
-  {
-    label: 'Edit',
-    submenu: [
-      { role: 'undo' },
-      { role: 'redo' },
-      { type: 'separator' },
-      { role: 'selectAll' },
-      { role: 'cut' },
-      { role: 'copy' },
-      { role: 'paste' },
-      { role: 'delete' },
-    ]
-  },
-  {
-    label: 'View',
-    submenu: [
-      { role: 'resetZoom' },
-      { role: 'zoomIn' },
-      { role: 'zoomOut' },
-      { type: 'separator' },
-      { role: 'togglefullscreen' }
-    ]
-  },
-  // ,
-  // {
-  //   role: 'help',
-  //   submenu: [
-  //     {
-  //       label: 'User Guide',
-  //       click () { require('electron').shell.openExternal('https://catcher-org.github.io/'); }
-  //     }
-  //   ]
-  // }
-];
-
 if (isDevMode) {
```

Comment:
There is some additional menu options created, if the app runs in developer mode.
Let's move this logic into the menu definition module.
(the module can export a function which accepts `isDevMode`)